src/main/webapp/WEB-INF/web.xml

```diff
@@ -83,4 +103,24 @@
     <error-code>500</error-code>
     <location>/WEB-INF/pages/error.jsp</location>
   </error-page>
+
+<error-page>
+  <error-code>404</error-code>
+  <location>/WEB-INF/pages/errorPageNotFound.jsp</location>
+</error-page>
+
+  <error-page>
+    <exception-type>com.es.phoneshop.model.order.OrderNotFoundException</exception-type>
+    <location>/WEB-INF/pages/errorOrderNotFound.jsp</location>
+  </error-page>
+
+  <filter>
```

Comment:
define filter before error pages, it's best practice to make your web.xml readable